parsec/api/protocol/organization.py

```diff
@@ -65,6 +65,8 @@ class OrganizationStatsRepSchema(BaseRepSchema):
     data_size = fields.Integer(required=True)
     metadata_size = fields.Integer(required=True)
     users = fields.Integer(required=True)
+    active_users = fields.Integer(required=True)
+    users_per_profile_detail = fields.Dict(required=True)
```

Comment:
`Dict` field is too broad, we should better specify the structure:
```python
class UsersPerProfileDetailItem(BaseSchema):
    profile = UserProfileField(required=True)
    active = fields.Integer(required=True)
    revoked = fields.Integer(required=True)

[...]
users_per_profile_detail = fields.List(UsersPerProfileDetailItem, required=True)
```

The rational for using a list of structs here is it's complex to define a dict with only user profile values as keys 

Having the profile value within the struct also allows for specialization (for instance we can have a `UserPerAdminProfileDetail` which define additional fields compared to the default `UserPerProfileDetail`).


Another solution would be to define a `Nested` class with each user profile as field, this would have the added benefit over the List approach of guaranteeing they won't be any user profile duplication:

```python
class UsersPerProfileDetailItem(BaseSchema):
    profile = UserProfileField(required=True)
    active = fields.Integer(required=True)
    revoked = fields.Integer(required=True)

class UsersPerProfileDetail(BaseSchema):
    ADMIN = fields.Nested(UsersPerProfileDetailItem, required=True)
    STANDARD = fields.Nested(UsersPerProfileDetailItem, required=True)
    OUTSIDER = fields.Nested(UsersPerProfileDetailItem, required=True)

[...]
users_per_profile_detail = fields.Nested(UsersPerProfileDetail, required=True)

```

However with this approach the returned `users_per_profile_detail` dict keys are `str` instead of `UserProfile` which is unexpected.

So in the end I prefer the list-based approach

@vxgmichel what do you think ?